amazon/Dockerfile

```diff
@@ -55,7 +55,8 @@ RUN set -ex \
   # NGINX Javascript module needed for APIM
   && yum update && yum -y install nginx-plus nginx-plus-module-njs  \
   # Install Controller Agent
-  && curl -k -sS -L ${CONTROLLER_URL}/install/controller/ > install.sh \
+  && curl -k -sS -L ${CONTROLLER_URL}/ > install.sh \
```

Comment:
I think the trailing `/` is not needed, so might be a good idea to remove it from here and other Dockerfiles

Reply:
Resolved.